# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/extra-cmake-modules/hotfix-findepoxy.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

# Findepoxy: use epoxy_INCLUDE_DIRS when looking for epoxy/glx.h

# The glx.h lookup hints at epoxy_INCLUDE_DIR (singular), which is never
# set; epoxy_INCLUDE_DIRS from the find_path above is correct. Without this
# epoxy_HAS_GLX stays FALSE when epoxy is installed outside /usr/include.

--- extra-cmake-modules-6.28.0/find-modules/Findepoxy.cmake.vanilla	2026-07-31 19:08:01.401164538 +0300
+++ extra-cmake-modules-6.28.0/find-modules/Findepoxy.cmake	2026-07-31 19:08:34.637508517 +0300
@@ -53,7 +53,7 @@
     NAMES epoxy
     HINTS ${PKG_epoxy_LIBDIR} ${PKG_epoxy_LIBRARY_DIRS}
 )
-find_file(epoxy_GLX_HEADER NAMES epoxy/glx.h HINTS ${epoxy_INCLUDE_DIR})
+find_file(epoxy_GLX_HEADER NAMES epoxy/glx.h HINTS ${epoxy_INCLUDE_DIRS})
 
 if (epoxy_GLX_HEADER STREQUAL "epoxy_GLX_HEADER-NOTFOUND")
     set(epoxy_HAS_GLX FALSE CACHE BOOL "whether glx is available")

